glossary.md

```diff
@@ -156,12 +243,54 @@ Schedulers are components in PD that generate scheduling tasks. Each scheduler i
 - `hot-region-scheduler`: Balances the distribution of hot Regions
 - `evict-leader-{store-id}`: Evicts all leaders of a node (often used for rolling upgrades)
 
+### Security Enhanced Mode (SEM)
+
+The Security Enhanced Mode (SEM) is used for finer-grained permission control of TiDB administrators. SEM is inspired by the design of systems such as [Security-Enhanced Linux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux). It reduces the abilities of users with the MySQL `SUPER` privilege and instead requires `RESTRICTED` fine-grained privileges to be granted as a replacement.
+
+For details, see [System Variables documentation - `tidb_enable_enhanced_security`](/system-variables.md#tidb_enable_enhanced_security).
+
+### Stale Read
+
+Stale Read is a mechanism that TiDB applies to read historical versions of data stored in TiDB. Using this mechanism, you can read the corresponding historical data of a specific point in time or within a specified time range, and thus save the latency brought by data replication between storage nodes. When you are using Stale Read, TiDB will randomly select a replica for data reading, which means that all replicas are available for data reading.
+
+For details, see [Stale Read](/stale-read.md).
+
 ### Store
 
 A store refers to the storage node in the TiKV cluster (an instance of `tikv-server`). Each store has a corresponding TiKV instance.
 
 ## T
 
+### Temporary table
+
+Temporary tables solve the issue of temporarily storing the intermediate results of an application, which frees you from frequently creating and dropping tables. You can store the intermediate calculation data in temporary tables. When the intermediate data is no longer needed, TiDB automatically cleans up and recycles the temporary tables. This avoids user applications being too complicated, reduces table management overhead, and improves performance.
+
+For details, see [Temporary Tables](/temporary-tables.md).
+
+### TiCDC
+
+[TiCDC](/ticdc/ticdc-overview.md) is a tool for incrementally replicating data in TiDB. It pulls the data change logs from the upstream TiKV and parses them into ordered row-level change data, which it then outputs to the downstream. For more information about the concepts and terms of TiCDC, see [TiCDC Glossary](/ticdc/ticdc-glossary.md).
+
+### TiDB Data Migration (DM)
+
+[TiDB Data Migration](https://github.com/pingcap/tiflow/tree/master/dm) (DM) is a convenient data migration tool that supports full data migration and incremental data replication from databases compatible with the MySQL protocol (MySQL, MariaDB, Aurora MySQL) to TiDB. Using DM helps simplify the data migration process and reduce operational costs.
+
+For more information about the concepts and terms of DM, see [TiDB Data Migration Glossary](/dm/dm-glossary.md).
+
+### TiDB Lightning
```

Comment:
```suggestion
### Lightning
```

Not sure if we should call it `TiDB Lightning` or `Lightning`...

Reply:
According to marketing guidelines (updated years ago), the product name is specifically `TiDB Lightning`. The same is true for DM. I'm not sure if there's any change to that, but I'll look into it.

Reply:
I'm fine with both. We could choose one of the two versions or add both "Lightning" and "TiDB Lightning" and then for the "Lightning" entry have a simple "See TiDB Lightning". 

Personally I think having a single entry would be best. I don't have a strong preference about which one it should be.